logger.go

```diff
@@ -46,6 +47,30 @@ var defaultProductionConfig = zap.Config{
 	ErrorOutputPaths:  []string{"stderr"},
 }
 
+// from https://github.com/uber-go/zap/blob/2314926ec34c23ee21f3dd4399438469668f8097/config.go#L98
+// but disable stacktraces.
+var defaultProductionGCPCloudConfig = zap.Config{
```

Comment:
maybe call this a cloud config, not gcp?

Reply:
Its specific to how Cloud Monitoring processes the fields for msg, severity etc... So i'm not sure generic Cloud is valid.

Reply:
okay cool

Reply:
sorry future eric for adding google stuff